lib/models/github-user.js

```diff
@@ -20,6 +20,8 @@ module.exports = (sequelize, DataTypes) => {
       type: DataTypes.STRING,
       allowNull: false,
     }),
+
```

Comment:
Should we add a note here saying for what reason we store this value? Or mention that this isn't used in the application and only intended for debugging?

Reply:
Makes sense. Done